examples/DeterministicSamplingLCDS2.m

```diff
@@ -0,0 +1,198 @@
+%% DeterministicSamplingLCDS2
+
+% Neat examples for on-sphere (S2) LCD sampling vor various spherical distributions. 
+% Created by Daniel Frisch at ISAS for libDirectional, Nov. 2019 
+
+% https://www.overleaf.com/read/bjchtykkytdf
```

Comment:
Are you sure you want to share the source code instead of simply the paper itself?